explainaboard/analysis/case.py

```diff
@@ -80,6 +81,37 @@ class AnalysisCaseLabeledSpan(AnalysisCaseSpan):
     predicted_label: str
 
 
+@dataclass
+class AnalysisCaseBlock(AnalysisCase):
+
+    text: str
+    orig_str: str
+
+
+@dataclass
+class AnalysisCaseLabeledBlock(AnalysisCaseBlock):
+    """
+    A bucket case that highlights a span in text along with a label.
```

Comment:
This docstring is not complete so I wasn't able to understand why we needed all the new variables here.